spring-rabbit-junit/src/main/java/org/springframework/amqp/rabbit/junit/BrokerRunning.java

```diff
@@ -614,4 +611,14 @@ private void closeResources(Connection connection, Channel channel) {
 		}
 	}
 
+	private static class BrokerNotAliveException extends RuntimeException {
```

Comment:
You throw this one from `public isUp()` method, so, there is not going to a simple way to analyze such a `private` exception. What does hold you back to have it as `public`?